packages/web/src/components/molecules/AgendaCard/TerminatedAgendaCard.tsx

```diff
@@ -71,35 +117,44 @@ export const TerminatedAgendaCard: React.FC<Props> = ({ agenda }) => {
             </span>
           </div>
           <VoteParticipate
+            named={agenda.type.named}
             voted={agenda.voters.voted}
             total={agenda.voters.total}
           />
           <VoteResult
-            type={agendaTags.public}
+            type={agenda.type.public}
             clickHandler={switchRevealChoice}
             revealChoice={revealChoice}
             voted={agenda.user.voted != null}
           />
-          <div css={[column, gap(6), w("fill")]}>
-            {agenda.choices.map(choice => (
-              <OptionVoteResult
-                name={choice.name}
-                count={choice.count}
-                totalCount={totalCount}
-                userChoice={revealChoice && agenda.user.voted === choice.id}
-              />
-            ))}
-          </div>
+          <div css={[column, gap(12), w("fill")]}>{optionVoteResult()}</div>
           <Divider />
-          <VoteDetail type={agendaTags.identified} />
+          <VoteDetail named={agenda.type.named} />
+          {agenda.type.named ? (
+            <div css={[column, gap(6), w("fill")]}>
+              {agenda.choices.map(choice => (
+                <VotedMembers
+                  userList={(agenda.voters.voted as Array<Voter>)
+                    .filter(voter => {
+                      if (choice.id === voter.choiceId) {
+                        return true;
+                      }
+                      return false;
+                    })
+                    .map(voter => voter.displayName)}
+                  name={choice.name}
+                />
+              ))}
+            </div>
+          ) : null}
```

Comment:
갑자기 든 생각인데, `비공개`이면서 `기명`인 경우가 존재하나요? `기명` 결과를 보여주면 결국 `공개`랑 다를 바가 없어서 🤔  이런 경우에는 일단 결과에서 `기명` 파트를 안 보여주는 게 맞는 것 같아요. 만약 admin에게도 이름이 보여지지 않아야 한다면 아예 `비공개`이면서 `기명`인 선택지를 제공하지 않아야 할 것 같구요!

<img width="312" alt="image" src="https://github.com/sparcs-kaist/biseo/assets/100757008/cac7c6d4-01af-46ed-9ff9-5607a8416c51">

```suggestion
         {agenda.type.public && (
           <>
              <VoteDetail named={agenda.type.named} />
              {agenda.type.named ? (
                ...
              ) : null}
            </>
```

Reply:
처리 방법이 여러가지일 것 같아서 (아예 불가능 하게 하기 / 관리자에게만 보이게 하기 / 그냥 놔두기 ..) 이슈로 따로 열어두겠습니다!